Minutes for Team meeting on March 13

vnajack opened the pull request.

vnajack committed:
Merge branch 'development' into team_meeting_minutes

vnajack committed:
Merge branch 'team_meeting_minutes' of https://github.com/speaking-portal-project-team-a/The-Speaking-Portal-Project into team_meeting_minutes

vnajack committed:
create minutes for team meeting on March 13

- updated branch
- wrote minutes

vnajack committed:
got rid of an extra #

vnajack committed:
Merge branch 'development' into team_meeting_minutes

vnajack committed:
create team minutes for March 20

- minutes template for Monday in case I can't attend due to traveling

vnajack committed:
Revert "create team minutes for March 20"

This reverts commit 816e1581467858bc214141dc003ab68ade78e941.

edouarde1 reviewed (approved):
Thanks for the minutes!

edouarde1 committed:
Merge branch 'development' into team_meeting_minutes

vnajack committed:
create minutes for team meeting on March 20

meeting minute template for Monday's meeting just in case I don't have service on the ferry

vnajack commented:
Added minutes template for March 20th just in case I don't have a signal on the ferry back

Commit:
meeting minutes

LoganParker commented on docs/meetings/team/2023_03_13.md:
```diff
@@ -0,0 +1,74 @@
+
+## 🃏 Joke of the Day
+
+On my first day at astronaut training, I vomited and asked the instructor, "Is this normal?"
+
+He said, "Not during a written exam, no."
```
I like this one, I don't remember it!

LoganParker commented on docs/meetings/team/2023_03_20.md:
```diff
@@ -0,0 +1,68 @@
+
+## 🃏 Joke of the Day
+
+There are two kinds of people in this world:
+
+1. Those who can extrapolate meaning from incomplete data
```
I remember this one!

LoganParker commented on docs/meetings/team/2023_03_20.md:
```diff
@@ -0,0 +1,68 @@
+- How did peer testing go?
+- Results from Google Form (Veronica had one session that had 2 forms submitted, sorry!)
+- Overall went well for all of us
+- FileNotFound Error:
+  - Veronica moved back a directory and did an `npm install`
+- Character default Error:
+  - When user types in invalid avatar it does not default
```
I believe these were the same errors, but I'm not sure so if we can replicate your filenotfound error that would be great

vnajack replied:
I'm really not sure why it was produced. I tried so many different solutions and that fix was the only one that made a difference. It might've just been a fluke because I haven't had issues since.

LoganParker reviewed (approved):
Awesome!! Thank you for the ⏳'s Veronica!

vnajack reviewed (commented):
Thank you, Ed, for taking the minutes for March 20th!
